spec/index.spec.coffee

```diff
@@ -516,9 +516,9 @@ describe 'payment', ->
       Payment.formatCardExpiry(expiry)
 
       ev = document.createEvent "HTMLEvents"
-      ev.initEvent "keypress", true, true
-      ev.eventName = "keypress"
-      ev.which = "/".charCodeAt(0)
+      ev.initEvent "beforeinput", true, true
+      ev.eventName = "beforeinput"
+      ev.data = "/"
```

Comment:
Let's replicate the tests so that we cover both `keypress` and `beforeinput`

Reply:
Done.